src/collections/Hero.tsx

```diff
@@ -103,22 +115,32 @@ export const Hero = ({
         theme ? theme : 'dark',
         className
       )}
-      style={{backgroundColor: bgColor, backgroundImage: `url(${bgImageOutput})`}}
+      style={{backgroundColor: bgColor}}
       aria-labelledby={HeroHeadingId}
     >
+      {bgImageOutput && (
+        <img
+          alt={parsedImageInfos?.alt}
+          srcSet={srcSet}
+          sizes="(min-width: 768w) 768px, (min-width: 1024w) 1024px, (min-width: 1440w) 1440px, 100vw"
```

Comment:
I'm surprised to see a hard-coded string here… Can't we get this string from a reusable place?